ipc/ipc_message_utils.cc

```diff
@@ -983,6 +1000,17 @@ bool ParamTraits<base::subtle::PlatformSharedMemoryRegion>::Read(
       return false;
     }
   }
+#if defined(CASTANETS)
+  if (static_cast<internal::PlatformFileAttachment*>(attachment.get())->file()
+      == -1) {
```

Comment:
@hyowon Could kCastanetsHandle defined in mojo/public/c/system/platform_handle.h be used instead of -1?

Reply:
https://github.com/Samsung/Castanets/pull/99#discussion_r297466285